lib/rules/no-magic-numbers.js

```diff
@@ -176,18 +193,19 @@ module.exports = {
                     raw = node.raw;
                 }
 
+                const parent = fullNumberNode.parent;
+
                 // Always allow radix arguments and JSX props
                 if (
                     isIgnoredValue(value) ||
+                    isIgnoredDefaultValue(fullNumberNode) ||
```

Comment:
Can you pull the check for whether the `ignoreDefaultValues` option is enabled out of `isIgnoredDefaultValue()` and put it here instead and then rename `isIgnoredDefaultValue()` to `isDefaultValue()`? That small refactor to `(ignoreDefaultValues && isDefaultValue(fullNumberNode))` should help future readers of this code by making the implementations for `ignoreArrayIndexes` and `ignoreDefaultValues` very similar to each other. Doing the enabled check here also avoids unnecessary function calls if the option is disabled.

Reply:
Done.